Russia

yatil opened the pull request:
Sorry for the somewhat messy Pull Request, will fix #373.

The (main) file that was changed is here: https://github.com/w3c/wai-policies-prototype/pull/379/files#diff-b21ec77cf6ae0e9b71546839eb239b3d

The preview is here: https://deploy-preview-379--wai-policies-prototype.netlify.com/policies/russia/

yatil commented:
Deploy preview for *wai-policies-prototype* processing.

Building with commit a3c5d73b82f4ba4224f6f9670e59e271088c2964

https://app.netlify.com/sites/wai-policies-prototype/deploys/5dd7da3d06cd420007669031

AndrewArch commented:
my error - we should have included the EN name for the policy too
"GOST R 52872-2012 Internet resources. Accessibility requirements for the visually impaired"

yatil commented:
@AndrewArch I updated the English title accordingly.

AndrewArch commented:
Thanks Eric - I think we're good to go

yatil commented:
@slhenry I addressed your comments in https://github.com/w3c/wai-policies-prototype/pull/379/commits/858aae05834c5ad20c11001933c76f4f80edd509 – please let me know when ready to publish.

AndrewArch commented:
Eric, thanks for modifications from Shawn and me. Good to go now pending Judy's ok

yatil commented:
@slhenry Is this now OK to publish?

slhenry commented:
I'm still struggling with pointing to such inaccessible and un-usable content. I'm thinking about putting the PDF link with the first link, and how to word it. Suggestions welcome.

slhenry commented:
Update: I talked to Judy a few weeks ago and she is even more uncomfortable than I am linking to such inaccessible info. She suggested contacting them and asking them to provide the info in accessible format. I'm not sure that we have direct contact?

We ran out of time before figuring out another approach, and have not been able to get back to it (and won't for a while). Sorry.

yatil commented:
The disclaimer language Judy came up with is now:

> <p>The laws and/or policies linked below have been submitted by the community, and have not been reviewed for accessibility. In the event that a law or policy may have accessibility problems, W3C/WAI welcomes information on more accessible versions of the linked laws/policies listed via our <a href="https://www.w3.org/WAI/policies/submission/">submission form</a>. We will also retain a link to the official version for each country.</p>

I hope this gets consensus. @AndrewArch, @slhenry?

AndrewArch commented:
Hi Eric and Shawn,

This sounds like a suitable disclaimer that still allows applicable
laws/polices to be listed.

Thanks.  Andrew

----------------
Andrew Arch
  Woodend, Australia



On Wed, 12 Feb 2020 at 23:02, Eric Eggert <notifications@github.com> wrote:

> The disclaimer language Judy came up with is now:
>
> The laws and/or policies linked below have been submitted by the
> community, and have not been reviewed for accessibility. In the event that
> a law or policy may have accessibility problems, W3C/WAI welcomes
> information on more accessible versions of the linked laws/policies listed
> via our submission form <https://www.w3.org/WAI/policies/submission/>. We
> will also retain a link to the official version for each country.
>
> I hope this gets consensus. @AndrewArch <https://github.com/AndrewArch>,
> @slhenry <https://github.com/slhenry>?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/w3c/wai-policies-prototype/pull/379?email_source=notifications&email_token=ABWOR3ZP3AHBZRQUBD7CLEDRCPQNLA5CNFSM4JQQIGDKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOELQQU4Q#issuecomment-585173618>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABWOR33PCV7CQQGKMTIVT43RCPQNLANCNFSM4JQQIGDA>
> .
>

iadawn commented on _layouts/policy.html:
```diff
@@ -14,18 +35,20 @@
 
     <div class="post-content">
-      {% if page.states %}<p>See below for information about <a href="#states">{{page.statesdesc.en}}</a>.</p>{% endif %}
+      <p>The laws and/or policies linked below have been submitted by the community, and have not been reviewed for accessibility. In the event that a law or policy may have accessibility problems, W3C/WAI welcomes information on more accessible versions of the linked laws/policies listed via our <a href="https://www.w3.org/WAI/policies/submission/">submission form</a>. We will also retain a link ...
```
@shawna-slh Would like to discuss this at next opportunity if we can

iadawn replied:
Move disclaimer to the footer for all pages

iadawn commented:
Update to Russia has been included other changes have been superceded
